lib/services/livesync/livesync-service.ts

```diff
@@ -117,26 +117,27 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 	}
 
 	private async refreshApplicationWithoutDebug(projectData: IProjectData, liveSyncResultInfo: ILiveSyncResultInfo, debugOpts?: IDebugOptions, outputPath?: string, settings?: IShouldSkipEmitLiveSyncNotification): Promise<void> {
-		const platformLiveSyncService = this.getLiveSyncService(liveSyncResultInfo.deviceAppData.platform);
+		const platform = liveSyncResultInfo.deviceAppData.platform;
+		const platformLiveSyncService = this.getLiveSyncService(platform);
 		try {
 			await platformLiveSyncService.refreshApplication(projectData, liveSyncResultInfo);
 		} catch (err) {
-			this.$logger.info(`Error while trying to start application ${projectData.projectId} on device ${liveSyncResultInfo.deviceAppData.device.deviceInfo.identifier}. Error is: ${err.message || err}`);
-			const msg = `Unable to start application ${projectData.projectId} on device ${liveSyncResultInfo.deviceAppData.device.deviceInfo.identifier}. Try starting it manually.`;
+			this.$logger.info(`Error while trying to start application ${projectData.projectIdentifiers[platform.toLowerCase()]} on device ${liveSyncResultInfo.deviceAppData.device.deviceInfo.identifier}. Error is: ${err.message || err}`);
+			const msg = `Unable to start application ${projectData.projectIdentifiers[platform.toLowerCase()]} on device ${liveSyncResultInfo.deviceAppData.device.deviceInfo.identifier}. Try starting it manually.`;
 			this.$logger.warn(msg);
 			if (!settings || !settings.shouldSkipEmitLiveSyncNotification) {
-				this.emit(LiveSyncEvents.liveSyncNotification, {
+				this.emitLivesyncEvent(LiveSyncEvents.liveSyncNotification, {
 					projectDir: projectData.projectDir,
-					applicationIdentifier: projectData.projectId,
+					applicationIdentifier: projectData.projectIdentifiers[platform.toLowerCase()],
```

Comment:
you can extract the applicationIdentifier in a variable above and use it everywhere
```TypeScript
const applicationIdentifier = projectData.projectIdentifiers[platform.toLowerCase()]
```